docs/dev/run_the_etl.rst

```diff
@@ -61,29 +65,29 @@ for one state (Idaho!) which takes around 20 minutes to process.
 
 Additional Notes
 ----------------
-These commands should result in a bunch of Python :mod:`logging` outputs,
-describing what the script is doing, and outputs in the ``sqlite``,
-``datapkg``, and ``parquet`` directories within your workspace. In particular,
-you should see new files at ``sqlite/ferc1.sqlite`` and ``sqlite/pudl.sqlite``,
-and a new directory at ``datapkg/pudl-fast`` or ``datapkg/pudl-full``
-containing several datapackage directories, one for each of the ``ferc1``,
-``eia`` (Forms 860 and 923), and ``epacems-eia`` datasets.
-
-Each of the data packages which are part of the bundle have metadata describing
-their structure, stored in a file called ``datapackage.json`` The data itself
-is stored in a bunch of CSV files (some of which may be :mod:`gzip` compressed)
-in the ``data/`` directories of each data package.
-
-You can use the ``pudl_etl`` script to process more or different data by
-copying and editing either of the the setting files, and running the script
-again with your new settings file as an argument. Comments in the example
-settings file explain the available parameters. Know that these example files
-are the only configurations that are tested and supported.
-
-If you want to re-run ``pudl_etl`` and replace an existing bundle of data
-packages, you can use ``--clobber``. If you want to generate a new data
-packages with a new or modified settings file, you can change the name of the
-output datapackage bundle in the configuration file.
-
-All of the  PUDL scripts have help messages if you want additional information
-(run ``script_name --help``).
+These commands should result in a bunch of Python :mod:`logging` outputs, describing
+what the script is doing, and outputs in the ``sqlite``, ``datapkg``, and ``parquet``
+directories within your workspace. When the ETL is complete you should see new files
+at ``sqlite/ferc1.sqlite`` and ``sqlite/pudl.sqlite``, and a new directory at
+``datapkg/pudl-fast`` or ``datapkg/pudl-full`` containing several datapackage
+directories, one for each of the ``ferc1``, ``eia`` (Forms 860 and 923), and
+``epacems-eia`` datasets.
+
+Each of the data packages which are part of the bundle have metadata describing their
+structure, stored in a file called ``datapackage.json`` The data itself is stored in
```

Comment:
I think it would sound better if you said "that are part" instead of "which". 

Remove comma after "structure". I might even put a period here and start a new sentence saying "The metadata are stored in a file called....." Also, in the next sentence, should it be "the data are stored" vs "is stored" ?